packages/frontend/src/ui/pool-remote-logo/index.tsx

```diff
@@ -20,13 +20,19 @@ export function PoolRemoteLogo({
     className,
 }: PoolRemoteLogoProps) {
     return (
-        <div
-            className={classNames(className?.root, styles.root, {
-                [styles[size]]: true,
-            })}
-        >
-            <RemoteLogo chain={chain} size={size} {...token1} defaultText=" " />
-            <RemoteLogo chain={chain} size={size} {...token0} defaultText=" " />
+        <div className={`${styles.root} ${styles[size]} ${className?.root}`}>
+            <RemoteLogo
+                chain={chain}
+                size={size}
+                {...token1}
+                defaultText={token0?.defaultText}
+            />
+            <RemoteLogo
+                chain={chain}
+                size={size}
+                {...token0}
```

Comment:
Here we are mixing the tokens, also we could use the `PoolRemoteLogo` component for this.

Reply:
Wait, we are inside the `PoolRemoteLogo` component itself. Anyway, I've addressed the comment in 16addf02c5a29363d358321ab31bbd09d3976d49